ert3/console/_console.py

```diff
@@ -103,7 +103,7 @@ def main():
     workspace = ert3.workspace.load(Path.cwd())
 
     if workspace is None:
-        sys.exit("Not inside an ERT workspace")
+        raise ValueError(f"Not inside an ERT workspace.")
```

Comment:
This is probably not what you want to do, because this makes the user feedback go from:
```bash
ert3 run something
Not inside an ERT workspace
```

to:
```bash
ert3 run something
Traceback (most recent call last):
  File ".../envs/ert3/bin/ert3", line 33, in <module>
    sys.exit(load_entry_point('ert', 'console_scripts', 'ert3')())
  .../_console.py", line 106, in main
    raise ValueError("Not inside an ERT workspace")
ValueError: Not inside an ERT workspace
```

What you want is to handle the different errors in `main` so we can exit gracefully (like the first example) with some helpful message.

Reply:
I think this could remain a `sys.exit`. In particular, `sys.exit` in the console is fine 👍